### ISSUE TITLE
Update Contribution Guide

### ISSUE THREAD

**Issue by 1adamf** [AntiFaffStrips/Strips#35]
# Problem Statement

Update the contribution guide to cover new organisation features

# Summary of Issue/Change

More to be added...
  
# Files effected

.github/Contribution Guide.md

**Comment by LEWISC44** [comment]
So i just add this to the .md file? @1adamf

**Comment by 1adamf** [comment]
No, Basically update the .md guide to follow the workflow there. That's just a rough guide of what order stuff should be done in but if you could turn that into an actual document and try and make it a bit more concise to match the style of the current contribution guide. Also some of these steps are already in the contribution guide so they won't need to be added
